core/App/screens/ProofDetails.tsx

```diff
@@ -83,37 +105,55 @@ const VerifiedProof: React.FC<VerifiedProofProps> = ({ record, navigation, isHis
     }
   }, [navigation])
 
+  useEffect(() => {
+    if (!connection) return
+    navigation.setOptions({ title: connectionLabel }) //
+  }, [connection])
+
   return (
-    <View>
-      <View style={styles.header}>
-        <View style={styles.headerTitleContainer}>
-          <CheckInCircle {...{ height: 45, width: 45 }} />
-          <Text style={styles.headerTitle}>{t('Verifier.InformationReceived')}</Text>
+    <View style={styles.container}>
+      {!isHistory && (
+        <View style={styles.header}>
+          <View style={styles.headerTitleContainer}>
+            <CheckInCircle {...{ height: 45, width: 45 }} />
+            <Text style={styles.headerTitle}>{t('Verifier.InformationReceived')}</Text>
+          </View>
+          <Text style={styles.headerDetails}>{t('Verifier.InformationReceivedDetails')}</Text>
         </View>
-        <Text style={styles.headerDetails}>{t('Verifier.InformationReceivedDetails')}</Text>
+      )}
+
```

Comment:
Can we create out custrom `Collapsible` componenton top and move related lines there?